requirements/gateway-requirements.yaml

```
@@ -25,3 +25,8 @@ watchfiles:
 aiohttp:
   pip_release: aiohttp
   max_major_version: 3
+
+boto3:
+  pip_release: boto3
+  minimum: "1.28.56"
+  max_major_version: 2
```

```suggestion
  max_major_version: 1
```

Sorry that this is a little confusing. It's an inclusive check. 
`max_major_version: 2` translates to `<3`